source/hardware/controllers/numark_ns6ii.rst

```diff
@@ -0,0 +1,543 @@
+.. _numark-ns6ii:
+
+Numark NS6II
+============
+
+.. sectionauthor::
+   Nikolaus Einhauser <nikolaus.einhauser@mixxx.org>
+
+
+.. figure:: ../../_static/controllers/numark_ns6ii_schematics/top_view.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Numark NS6II (schematic view)
+   :figclass: pretty-figures
+
+   Numark NS6II (schematic view)
+
+
+The Numark NS6II is an all-in-one USB :term:`MIDI` controller with an integrated audio interface.
+It has controls for 2 decks that can be toggled between decks to play with 4 decks.
+Its jogwheels feature a built-in screen highlighting basic information on the currently
+playing track of each deck.
+
+-  `Manufacturer’s product page <http://www.numark.com/product/ns6ii>`__
+-  `Serato DJ Hardware Page <https://serato.com/dj/hardware/numark-ns6ii>`__
+-  `User Guide <https://www.numark.com/images/product_downloads/NS6II-UserGuide-v1.1.pdf>`__
+
+.. versionadded:: 2.5.0
```

Comment:
```suggestion
.. versionadded:: 2.5.1
```

Reply:
right, that ship has sailed :sweat_smile: